### ISSUE TITLE
Make a toggle setting for the physgun to let the object react to the world (collission) while changing its position

### ISSUE THREAD

**Issue by Sebihunter** [nanos-world/nanos-world-sandbox#2]
It can help a lot to properly align objects before welding them to eachother

**Comment by gtnardy** [comment]
Yeah at first I tried doing like that, but just 'setting' the objects location doesn't interact with physics, I may need to find another solution which sends the movement to the physics thread 🤔
